### PR TITLE
fix: using javascript page not found

### LINKED ISSUE #1692: Javascript API usage page 404
404 not found for this page : referenced in main menu

https://developer.mozilla.org/fr/docs/Mozilla/Add-ons/WebExtensions/Using_the_JavaScript_APIs

### PR DESCRIPTION
**Pull request by mattlavio**
This PR fixes https://github.com/mdn/translated-content/issues/1692

I went ahead and copied what the japan version does for `_redirects.txt` file and applied the new page to the other references in French

### CODE REVIEW

File: files/fr/_redirects.txt [hunk 1]
```diff
@@ -3103,6 +3103,7 @@
 /fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/elements_menu_contextuel	/fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/Context_menu_items
 /fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/pages_web_incluses	/fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/Extension_pages
 /fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/panneaux_devtools	/fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/devtools_panels
+/fr/docs/Mozilla/Add-ons/WebExtensions/Using_the_JavaScript_APIs	/fr/docs/Mozilla/Add-ons/WebExtensions/API
```
Comment (m4thieulavoie) [inline files/fr/_redirects.txt:3106]:
```suggestion
/fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/panneaux_devtools	/fr/docs/Mozilla/Add-ons/WebExtensions/user_interface/devtools_panels
/fr/docs/Mozilla/Add-ons/WebExtensions/Using_the_JavaScript_APIs	/fr/docs/Mozilla/Add-ons/WebExtensions/API
```

Comment (m4thieulavoie) [inline files/fr/_redirects.txt:3106]:
```suggestion
/fr/docs/Mozilla/Add-ons/WebExtensions/Using_the_JavaScript_APIs	/fr/docs/Mozilla/Add-ons/WebExtensions/API
```

**Comment by m4thieulavoie** [comment]
Not too sure why my redirect isn't passing in CI. I could use a cue, I'm sure I'm not that far to make it work. Worst case I can remove the line. Let me know!

**Comment by SphinxKnight** [comment]
@m4thieulavoie I'll be having a look in the coming days, I guess that's due to the redirect chain (A->B, B->C, etc.)

**Comment by SphinxKnight** [comment]
@m4thieulavoie I've used `yarn tool validate-redirects fr --strict` (from the mdn/yari repo) and `yarn content add-redirect` from mdn/content to "re-add" the redirect and it ended up earlier in the file.

I guess editing `_redirects.txt` is a bit risky when done manually :) Don't hesitate to ping/ask if needed.

**Review by SphinxKnight — COMMENTED** [review]
👍

**Comment by m4thieulavoie** [comment]
> @m4thieulavoie I've used `yarn tool validate-redirects fr --strict` (from the mdn/yari repo) and `yarn content add-redirect` from mdn/content to "re-add" the redirect and it ended up earlier in the file.
> 
> I guess editing `_redirects.txt` is a bit risky when done manually :) Don't hesitate to ping/ask if needed.

Awesome thanks for looking into it! :)

**Pull request merged**